web/src/engine/predictive-text/worker-thread/src/tests/mocha/cases/edit-distance/context-tokenization.js

```diff
@@ -64,17 +64,17 @@ describe('isSubstitutionAlignable', () => {
   it(`returns false:  'apple' => 'banana' (subs allowed)`, () => {
     // edit path:  'insert' ('b' of banana), 'match' (on leading a), rest are 'substitute'.
     // 1 match vs 4 substitute = no bueno.  It'd require too niche of a keyboard rule.
-    assert.isFalse(isSubstitutionAlignable('banana', 'apple', true));
+    assert.isFalse(isSubstitutionAlignable('banana', 'apple', { forNearCaret: true }));
```

Comment:
Whoops, that slipped in a bit early.  Fortunately, it is "truthy", so it actually works.